includes/Page_Template_Post_Type.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * Class Page_Template_Post_Type.
+ *
+ * @package   Google\Web_Templates
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+use Google\Web_Stories\REST_API\Stories_Page_Template_Controller;
+
+/**
+ * Class Page_Template_Post_Type.
+ */
+class Page_Template_Post_Type extends Service_Base {
+	/**
+	 * The slug of the page template post type.
+	 *
+	 * @var string
+	 */
+	const POST_TYPE_SLUG = 'web-story-page';
+
+	/**
+	 * Registers the post type for page templates.
+	 *
+	 * @since 1.6.0
+	 *
+	 * @return void
+	 */
+	public function register() {
+		$post_type_object = get_post_type_object( Story_Post_Type::POST_TYPE_SLUG );
+		$capabilities     = [];
+		if ( $post_type_object ) {
+			$capabilities = [
+				'edit_post'              => $post_type_object->cap->edit_posts,
+				'read_post'              => $post_type_object->cap->edit_posts,
+				'delete_post'            => $post_type_object->cap->delete_posts,
+				'edit_posts'             => $post_type_object->cap->edit_posts,
+				'edit_others_posts'      => $post_type_object->cap->edit_posts,
+				'delete_posts'           => $post_type_object->cap->delete_posts,
+				'publish_posts'          => $post_type_object->cap->edit_posts,
+				'read_private_posts'     => $post_type_object->cap->edit_posts,
+				'delete_private_posts'   => $post_type_object->cap->delete_posts,
+				'delete_published_posts' => $post_type_object->cap->delete_posts,
+				'delete_others_posts'    => $post_type_object->cap->delete_posts,
+				'edit_private_posts'     => $post_type_object->cap->edit_posts,
+				'edit_published_posts'   => $post_type_object->cap->edit_posts,
+				'create_posts'           => $post_type_object->cap->edit_posts,
+			];
```

Comment:
I mapped edit to create / edit / read and delete to delete. This made sense to me.